specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,702 @@
+            "schema": {
+              "$ref": "#/definitions/SessionPool"
+            }
+          },
+          "202": {
+            "description": "Session Pool update has been started.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Update Session Pool": {
+            "$ref": "./examples/SessionPools_Patch.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ContainerAppsSessionPools"
+        ],
+        "summary": "Delete a Session Pool.",
+        "description": "Delete the Session Pool with the given name.",
+        "operationId": "ContainerAppsSessionPools_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionPoolNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Session Pool delete has been started.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Session Pool does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Delete Session Pool": {
+            "$ref": "./examples/SessionPools_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScaleConfiguration": {
+      "description": "Scale configuration.",
+      "type": "object",
+      "properties": {
+        "maxConcurrentSessions": {
+          "description": "The maximum count of Sessions at the same time.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "readySessionInstances": {
```

Comment:
minReadySessionInstances ?

Reply:
There was a conversation about the naming of this property, and I think the conclusion was readySessionInstances, so I am using this here.

Reply:
Thanks, there are some changes need to be made in the server codebase in order to meet ARM requirements, which is still in progress, including Patch SessionPool and async Post GenerateSessions.